### PR TITLE
update: improve error message from new update server response code

### PR DESCRIPTION
**Pull request by paulpv**

### CODE REVIEW

**Comment by paulpv** [comment]
@Trouffman for your review, if you are able

**Comment by paulpv** [comment]
This has been ready for review for ~24 hours.

I would like to get this into 4.14.1.

File: src/forms/output-settings.cpp [hunk 1]
```diff
@@ -57,45 +57,70 @@ OutputSettings::OutputSettings(QWidget *parent)
...
+					// English only text is OK here, just like it is for all code comments.
+					errorText += R"(
+The update server says you are not using an official release.<br>
+<br>
+Update checks are only supported for official releases.
+)";
+					if (Config::LogDebug()) {
+						errorText += R"(<br>
+<br>
+If you are running a local build, don't forget to add your build info to the update server.
```
Comment (Trouffman) [inline src/forms/output-settings.cpp:115]: We will need to havethis process / step added to the wiki.

**Review by Trouffman — APPROVED** [review]

File: src/forms/output-settings.cpp (hunk 1 above)
Comment (paulpv) [reply src/forms/output-settings.cpp:115]: Documented at https://github.com/DistroAV/DistroAV/wiki/3.-Development#release_hashes

**Pull request merged**